testspec.yml

```diff
@@ -20,7 +20,7 @@ phases:
       - pip install -r test/requirements.txt
       - pip install scheduler/.
       - echo Running pytest $TEST_TYPE tests on $DLC_IMAGES...
-      - python test/testrunner.py
+      - export PYTHONPATH=$(PYTHONPATH):$(pwd)/src && python test/testrunner.py
```

Comment:
This code change is necessary, otherwise an error will be threw
```
 Traceback (most recent call last):
   File "test/testrunner.py", line 16, in <module>
     from test_utils import eks as eks_utils
   File "/codebuild/output/src011798628/src/github.com/aws/deep-learning-containers/test/test_utils/__init__.py", line 17, in <module>
     from src.config.test_config import ENABLE_BENCHMARK_DEV_MODE
 ModuleNotFoundError: No module named 'src'
```

Reply:
nit: does it have to be on the same line? Don't the env variables persist?

Reply:
It should be Okay to separate into 2 lines. I can take a try